cabby/__init__.py

```diff
@@ -20,13 +21,24 @@ def create_client(host=None, port=None, discovery_path=None, use_https=False,
     :param int port: TAXII server port
     :param str discovery_path: Discovery Service relative path
     :param bool use_https: if HTTPS should be used
+    :param string url: URL to infer host, port, discovery_path, and use_https
```

Comment:
if you're infering discovery path from it, maybe it's better to call it `discovery_url` instead of generic `url`?

Reply:
Good point. Fixed :+1: